crates/stages-api/src/metrics/listener.rs

```diff
@@ -45,8 +45,7 @@ pub enum MetricEvent {
 #[derive(Debug)]
 pub struct MetricsListener {
     events_rx: UnboundedReceiver<MetricEvent>,
-    /// underline metrics of stages
-    pub sync_metrics: SyncMetrics,
+    pub(crate) sync_metrics: SyncMetrics,
```

Comment:
@onbjerg The visibility change was introduced by `issue-7599`, at that time, I haven't extract pipeline to stages-api. So I have to change it back.